terraform/main.tf

```diff
@@ -0,0 +1,139 @@
+terraform {
+  backend "gcs" {
+    bucket = "f3441e415e6e5e7d-bucket-tfstate"
+    prefix = "terraform/state"
+  }
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "4.51.0"
+    }
+  }
+}
+
+variable "project_id" {
+  type    = string
+  default = "catalyst-cooperative-pudl"
+}
+
+variable "catalyst_people" {
+  type = list(string)
+  default = [
+    "trenton.bush@catalyst.coop",
+    "katherine.lamb@catalyst.coop",
+    "zach.schira@catalyst.coop",
+    "dazhong.xia@catalyst.coop"
+  ]
+}
+
+provider "google" {
+  project = var.project_id
+  region  = "us-east1"
+  zone    = "us-east1-c"
+}
+
+resource "google_service_account" "mozilla_dev_sa" {
+  account_id   = "mozilla-dev-sa"
+  display_name = "Mozilla dev"
+}
+
+resource "random_id" "bucket_prefix" {
+  byte_length = 8
+}
+
+resource "google_storage_bucket" "sec_10ks" {
+  name          = "${random_id.bucket_prefix.hex}-bucket-sec-10ks"
+  location      = "US"
+  storage_class = "STANDARD"
+  versioning {
+    enabled = true
+  }
+}
+
+resource "google_storage_bucket" "tfstate" {
+  name          = "${random_id.bucket_prefix.hex}-tfstate"
+  location      = "US"
+  storage_class = "STANDARD"
+  versioning {
+    enabled = true
+  }
+}
+
+resource "google_storage_bucket_iam_binding" "catalyst_gcs_access" {
+  bucket  = google_storage_bucket.sec_10ks.name
+  role    = "roles/storage.admin"
+  members = ["serviceAccount:${google_service_account.mozilla_dev_sa.email}"]
+}
+
+resource "google_project_iam_binding" "catalyst_cloudsql_instance_user" {
+  project = var.project_id
+  role    = "roles/cloudsql.instanceUser"
+  members = ["serviceAccount:${google_service_account.mozilla_dev_sa.email}"]
+}
+
+resource "google_project_iam_binding" "catalyst_cloudsql_client" {
+  project = var.project_id
+  role    = "roles/cloudsql.client"
+  members = ["serviceAccount:${google_service_account.mozilla_dev_sa.email}"]
+}
+
+resource "google_project_iam_binding" "catalyst_people_editors" {
```

Comment:
This also only makes sense if we set up Mozilla as a new project - otherwise we're just setting all these catalyst_people to Editor in the main project, which might not be what we want.